app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,48 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private List<Task> tasks = new ArrayList<>();
+    private Long newId = 0L;
+
+    @GetMapping
+    public List<Task> list() {
+        return tasks;
+    }
+
+    @GetMapping("/{id}")
+    public Task get(@PathVariable Long id) throws IOException {
+        Task task = findTask(id);
+        return task;
```

Comment:
바로 반환해도 될 것 같아요

```suggestion
        return findTask(id);
```

See also
  - https://refactoring.com/catalog/inlineVariable.html

Reply:
덕분에 또 하나 배워갑니다 ㅎㅎ